README.md

```diff
@@ -47,362 +43,20 @@ account file must be provided as a `Secret`.
-
-  ```bash
-  choco install kustomize
-  ```
-
-* [Kubectl 1.11.3+](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to
-deploy the operator on the cluster.
-  * Make sure
-  [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
-  is properly set up with your Kubernetes cluster data.
-
-  * *Unix/Linux* users with
-  [Snap](https://snapcraft.io/docs/installing-snapd):
-
-  ```bash
-  snap install kubectl --classic
-  ```
-
-  * *MacOs* users with [HomeBrew](https://brew.sh/):
-
-  ```bash
-  brew install kubectl
-  ```
-
-  * *Windows* users: follow
-  [this section](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows)
-  of the documentation
-
-* [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder#installation)
-in case you want to add controllers and other Kubernetes components.
-  * *MacOs* users with [HomeBrew](https://brew.sh/):
-
-  ```bash
-  brew install kubebuilder
-  ```
-
-  * *Other systems*: follow
-  [this section](https://book.kubebuilder.io/quick-start.html#installation)
-  on the documentation.
-
-The aforementioned components need to be present in your machine and need not
-to be necessarily installed on your Kubernetes worker nodes. Also please note
-that these components are only needed just for the building phase.
-
-### Deploy
-
-First, you need to build and push the docker image to your container registry
-of choice. To ease the process up, you can edit the `Makefile` - included in
-the root folder of the project - by entering the image repository where
-you want to push the image:
-
-```makefile
-IMG ?= example.com/username/image:tag
-```
-
-Make sure you are properly logged in your container registry of choice before
-proceeding. Most of the times, running `docker login <registry>` as documented
-[here](https://docs.docker.com/engine/reference/commandline/login/) should be
-enough, but we encorage you to read your container registry's official
-documentation to know how to do that.  
-Build and push the image:
-
-```bash
-make docker-build
-make docker-push
-```
-
-Deploy the operator on your cluster by running:
-
-```bash
-make deploy
-```
-
-The operator will be installed in one of the available and suitable nodes
-of your cluster.
+## Try It Out
 
-Please refrain from using docker commands directly as the described method
-will also test the program before building it.
+If you want to quickly see how CNWAN Operator works, you can follow this simple
+step by step [quickstart](./docs/quickstart.md) guide.
 
-## Removing from Your Cluster
+## Documentation
 
-To remove the operator from your Kubernetes cluster, navigate to the root
-directory of the project and execute:
+If you want to learn more about CNWAN Operator, you can follow the
+documentation inside the [docs](./docs) folder.
 
-```bash
-make remove
-```
+You can start by reading [Concepts](./docs/concepts.md) or try to deploy it
```

Comment:
Maybe instead of this sentence, can we have bullet points in the "Documentation" section that describe each of the files available in the ./docs folder?

Reply:
Just did, although I forgot to write description. I think though that the names are understandable. Let me know if you think it's better to change them though